### DISCUSSION TITLE
Option to see the list of all the people who viewed the document

### DISCUSSION THREAD

**Discussion by a2tm7a** [outline/outline#1556] [General]
![Screenshot 2020-09-17 at 6 44 10 PM](https://user-images.githubusercontent.com/10511458/93475040-c8665300-f915-11ea-9298-fa423fed9c7a.png)

Option to click on the +n and view the list of people who viewed it along the time last opened by them. 
1. Is it possible to do that?
2. Can we sort the list of people viewing the document in the order of ascending recency?

**Comment by matbrgz** [comment]
https://github.com/outline/outline/issues/1105

Sort is awesome idea.
